Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN gem install bundler -v 2.3.9
 
 COPY lib/uffizzi/version.rb ./lib/uffizzi/
 COPY uffizzi.gemspec .
-COPY Gemfile* .
+COPY Gemfile* ./
```

Comment:
It's unnecessary

Reply:
This is necessary when I build on my workstation. Without the foreslash, the build ends with this error:
```
Step 10/21 : COPY Gemfile* .
When using COPY with more than one source file, the destination must be a directory and end with a /
```